documentation/ame-guide/modules/ROOT/nav.adoc

```diff
@@ -9,6 +9,7 @@
 ** xref:edit-models.adoc[Edit models]
 ** xref:edit-elements.adoc[Edit elements]
 ** xref:navigate-models.adoc[Navigate models]
+** xref:shared-models.adoc[Shared models]
```

Comment:
In the shared-model-filename.png image, it's written as "Shared Model" (both words in capitalized). Is it correct to have a different capitalization here?